### PR TITLE
Update icons.yml

### PR DESCRIPTION
**Pull request by grvrg**
consolidate Viacoin formatting to be in line with formatting of Bitcoin and other currencies

### CODE REVIEW

File: src/icons.yml [hunk 1]
```diff
@@ -5256,12 +5256,15 @@ icons:
     categories:
       - Web Application Icons
 
-  - name:       Viacoin
+  - name:       Viacoin (VIA)
     id:         viacoin
     unicode:    f237
     created:    4.3
     url:        viacoin.org
+    aliases:
```
Comment (wolfy1339) [inline src/icons.yml]: This is not necessary

Comment (grvrg) [reply src/icons.yml]: Bitcoin is listed as **Bitcoin (BTC)**, other currencies are also listed with their short symbols such as **Euro (EUR)**

Comment (wolfy1339) [reply src/icons.yml]: I wasn't talking about the whole update, just that the alias is not needed as it's the same as the id

Comment (grvrg) [reply src/icons.yml]: i see :)
so it should be probably changed as follows?
id to **via** and alias to **viacoin**

**Comment by tagliala** [comment]
Thanks!

**Pull request merged**